plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -335,7 +335,7 @@ func (l *persistentVolumeLabel) getAzurePVLabeler() (cloudprovider.PVLabeler, er
 
 		azurePVLabeler, ok := cloudProvider.(cloudprovider.PVLabeler)
 		if !ok {
-			return nil, errors.New("Azure cloud provider does not implement PV labeling")
+			return nil, errors.New("azure cloud provider does not implement PV labeling")
```

Comment:
can be change to the following to preserve the capitalization of the cloud provider name:
> the Azure cloud provider...